mcs/class/referencesource/mscorlib/system/threading/threadpool.cs

```diff
@@ -675,7 +677,9 @@ public void Enqueue(IThreadPoolWorkItem callback, bool forceGlobal)
                     }
                 }
             }
-
+#if MONO
+            InternalPerformanceCounter.Increment(THREADPOOL_WORKITEMS);
```

Comment:
This is an icall on a fast path, no reason to do it.

Reply:
This will force a bucket of stuff to be brought in even when we don't need them.